src/pages/Search/SearchAdvancedFiltersPage/SearchFiltersHasPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SearchMultipleSelectionPicker from '@components/Search/SearchMultipleSelectionPicker';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as SearchActions from '@userActions/Search';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+type HasItem = {
```

Comment:
This name is super confusing because it sounds like a bool variable name.
I suggest simply `FilterItem`